pyscript.py

```diff
@@ -0,0 +1,19 @@
+"""
+A script to enforce the maximum line length of each rst file to 75 
+characters because this makes editing multiple files in split editor mode or in split 
+terminal vim windows much easier which in turns increases productivity.
+"""
```

Comment:
never use these generic names please, rename it something like "`check_line_length.py`".

Also add a she bang in the beginning of the file (`#!/usr/bin/env python`) and make it executable so we can call it like `./check_line_length.py` which is more consistent with what we do in different repositories.